src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -1061,7 +1061,11 @@ private void updateNameReference(
               Node assign = IR.assign(getProp, nameRef.getFirstChild().detachFromParent());
               assign.setJSDocInfo(info);
               Node expr = IR.exprResult(assign).useSourceInfoIfMissingFromForTree(nameRef);
-              parent.replaceWith(expr);
```

Comment:
Hmm probably, but it would take some hunting to find. The general idea is:

```java
if (parent.getChildCount() == 1) {
  // existing logic was correct
} else if (parent.getFirstChild() == nameRef) {
  // Insert the new expression right before the parent
} else if (nameRef.getNext() == null) {
  // Insert the new expression right after the parent
} else {
  Node newVar = parent.cloneNode();  // clone the var/let/const statement
  while (nameRef.getNext() != null) {
    newVar.addChildToBack(parent.removeChildAfter(nameRef))
  }
  parent.getParent().addChildAfter(newVar, parent);
  // Insert the new expr right after parent
}
```